src/event.rs

```diff
@@ -896,12 +987,6 @@ pub enum Force {
         /// The value of this field is sufficiently high to provide a wide
         /// dynamic range for values of the `force` field.
         max_possible_force: f64,
-        /// The altitude (in radians) of the stylus.
-        ///
-        /// A value of 0 radians indicates that the stylus is parallel to the
-        /// surface. The value of this property is Pi/2 when the stylus is
-        /// perpendicular to the surface.
-        altitude_angle: Option<f64>,
```

Comment:
Is there a reason that this got removed?

Reply:
Yes, because its pen input and it doesn't exist in this current API.

My plan was to re-introduce the same code back again in #3810.
But if you want to I'm happy to introduce pen input in this PR directly.

Reply:
No, that's fine, I didn't read your PR description properly, that's on me ;)